src/js/reducers/app-state-reducer.js

```diff
@@ -0,0 +1,80 @@
+import { ENABLE_SETTINGS, DISABLE_SETTINGS, TOGGLE_WIDGET, OPEN_WIDGET, CLOSE_WIDGET, SHOW_SETTINGS, HIDE_SETTINGS, SHOW_SETTINGS_NOTIFICATION, HIDE_SETTINGS_NOTIFICATION, SET_SERVER_URL, SET_EMAIL_READONLY, UNSET_EMAIL_READONLY, UPDATE_READ_TIMESTAMP } from 'actions/app-state-actions';
```

Comment:
`import * as AppStateActions from 'actions/app-state-actions'` and do `AppStateActions.UNSET_EMAIL_READONLY` everywhere.. the formatter won't let me put that on mutliple lines